### ISSUE TITLE
Confusing Communications List UI after re-subscribing

### ISSUE THREAD

**Issue by JimMichael** [SparkDevNetwork/Rock#3496]
### Description

When unsubscribing from a Communication List, a note stating _Unsubscribed_ is added to the record, but if the same person re-subscribes to the list, the note remains.

### Steps to Reproduce

1. Subscribe to a List
2. Unsubscribe from same List. 
3. Go to List and observe there is a note icon for the person with note of _Unsubscribed_
4. Subscribe to the List again and observe that the unsubscribed note still remains, but it's confusing because the person is actually subscribed.

**Expected behavior:**

Either re-subscribing should clear the note, OR perhaps the note should have a date so we'd see "unsubscribed 1/10/2019" and if someone subsequently re-subscribes the note would at least have some historical context.

**Actual behavior:**

The list says someone is unsubscribed when they are not (sure it's easy enough to see that they are active vs. inactive, but it's still a non-polished UI.)

### Versions

* **Rock Version:** v7+
![image](https://user-images.githubusercontent.com/3903408/50998450-05e1ce80-14ed-11e9-9afe-57c5022bc170.png)

**Comment by JimMichael** [comment]
Thanks, @nairdo! I just tried it again on our 8.3 system and it worked like you describe when I use the block to unsubscribe/re-subscribe... I think I got confused during process street testing and _manually_ re-activated someone who was unsubscribed, and in that case it leaves the note and I thought it was a bug. But in that case I think it's expected behavior as you're manually touching the list.

Sorry for the false alarm!
